projects/1.16/assets/tetra/tetra/lang/zh_cn.json

```
@@ -39,339 +39,340 @@
+  "tetra.tool.shovel.craft_requirement": "加工需要一个%d级锹",
+  "tetra.tool.shovel.material_requirement": "加工此材料的基本要求是一个%d级锹，此要求可能会因材料的用途不同而不同",
   "tetra.tool.hoe": "锄",
-  "tetra.tool.hoe.requirement": "此操作需要%d级的锄。",
-  "tetra.tool.hoe.craft_requirement": "加工需要一个%d级锄。",
-  "tetra.tool.hoe.material_requirement": "加工这种材料的基本要求是一个%d级锄，这个要求可能会根据材料的用途而改变。",
+  "tetra.tool.hoe.requirement": "此操作需要%d级锄",
+  "tetra.tool.hoe.craft_requirement": "加工需要一个%d级锄",
+  "tetra.tool.hoe.material_requirement": "加工此材料的基本要求是一个%d级锄，此要求可能会因材料的用途不同而不同",
   "tetra.tool.cut": "切削",
-  "tetra.tool.cut.requirement": "这一步需要一个%d级的切削工具。\n§7大部分的剑可用于切削。",
-  "tetra.tool.cut.craft_requirement": "加工需要一个%d级切削工具。",
-  "tetra.tool.cut.material_requirement": "加工这种材料的基本要求是一个%d级切削工具，这个要求可能会根据材料的用途而改变。\n§7大部分的剑可以被用于切削。",
+  "tetra.tool.cut.requirement": "这一步需要一个%d级切削工具\n§7大部分的剑可用于切削",
+  "tetra.tool.cut.craft_requirement": "加工需要一个%d级切削工具",
+  "tetra.tool.cut.material_requirement": "加工此材料的基本要求是一个%d级切削工具，此要求可能会因材料的用途不同而不同\n§7大部分的剑可以被用于切削",
   "tetra.tool.pry": "撬",
-  "tetra.tool.pry.requirement": "此操作需要%d级的撬",
-  "tetra.tool.pry.craft_requirement": "加工需要一个%d级撬。",
-  "tetra.tool.pry.material_requirement": "加工这种材料的基本要求是一个%d级撬，这个要求可能会根据材料的用途而改变。",
+  "tetra.tool.pry.requirement": "此操作需要%d级撬\n§7羊角头可用于撬",
+  "tetra.tool.pry.craft_requirement": "加工需要一个%d级撬",
+  "tetra.tool.pry.material_requirement": "加工此材料的基础要求是一个%d级撬，此要求可能会因材料的用途不同而不同\n§7羊角头可用于撬",
   "tetra.tool.available": "你有一个%d级的",
   "tetra.stats.efficiency_suffix": "效率",
   "tetra.stats.strength_suffix": "力量",
   "tetra.stats.tier_suffix": "等级",
   "tetra.stats.level_suffix": "等级",
   "tetra.stats.reach": "触及范围",
-  "tetra.stats.reach.tooltip": "可以触及最远§e%s§r格的方块和实体",
-  "tetra.stats.reach.tooltip_blocks": "此物品最多可以格挡§e%s§r格距离，它不影响实体能被击中的距离",
+  "tetra.stats.reach.tooltip": "可以触及最远§e%s§r格处的方块和最远§e%s§r格处的实体",
+  "tetra.stats.reach.tooltip_blocks": "可以触及最远§e%s§r格处的方块，触及实体的距离不受影响",
   "tetra.stats.reach_normalized": "触及范围",
-  "tetra.stats.reach_normalized.tooltip": "增加这个物品§e%s§r个方块的触及范围",
+  "tetra.stats.reach_normalized.tooltip": "触及范围改变量：§e%s§r格",
   "tetra.stats.durability": "耐久",
-  "tetra.stats.durability.tooltip": "这个物品有§e%s§r点耐久",
-  "tetra.stats.durability.tooltip_extended": "§7通常物品每次使用会消耗1或2点耐久，当耐久值达到0时，物品就会破损，必须在工作台上进行修复才能再次使用",
+  "tetra.stats.durability.tooltip": "此物品有§e%s§r点耐久",
+  "tetra.stats.durability.tooltip_extended": "§7通常物品每次使用会消耗1或2点耐久，当耐久降到0时，物品就会破损，必须在加工台上进行修复才能再次使用",
   "tetra.stats.speed": "速度",
-  "tetra.stats.speed.tooltip": "影响物品可以被挥动得多快\n \n§e%s §7秒的挥动间隔\n§e%s§7倍的采集效率",
-  "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击会造成更少的伤害。如横扫或急速打击等效果只有在攻击指示条完全充满时才会被触发。§7速度值相当于采集效率的倍率。取决于大于或小于1，它会加快或减缓方块被破坏的速度。倍率计算方式如下：§f§o倍率 = 速度 *0.5+0.5§r§7",
+  "tetra.stats.speed.tooltip": "影响物品的挥动速度\n \n§7挥动间隔§e%s§7秒\n§7采集效率§e%s§7倍",
+  "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击造成的伤害较少。如横扫或急速打击等效果只有在攻击指示条完全充满时才会被触发\n \n§7速度是采集效率的乘数，它会加快或减缓方块被破坏的速度。倍率计算方式如下：§f§o倍率 = 速度 * 0.5 + 0.5§r§7",

```

```suggestion
  "tetra.stats.speed.tooltip_extended": "挥动“冷却”由屏幕中央的指示条显示。可以在冷却完毕前挥动，但是攻击造成的伤害较少。如横扫或急速打击等效果只有在攻击指示条完全充满时才会被触发\n \n§7速度是采集效率的乘数，它会加快或减缓方块被破坏的速度。倍率计算方式如下：§f§o倍率=速度*0.5+0.5§r§7",

```
个人认为这里不需要空格，当然留着也无伤大雅